packages/neon_framework/lib/src/utils/launch_url.dart

```diff
@@ -0,0 +1,10 @@
+import 'package:flutter/cupertino.dart';
+import 'package:neon_framework/models.dart';
+import 'package:neon_framework/utils.dart';
+import 'package:url_launcher/url_launcher.dart' as url_launcher;
+
+Future<bool> launchUrl(BuildContext context, String url) async {
+  final account = NeonProvider.of<Account>(context);
+  final uri = account.completeUri(Uri.parse(url));
+  return url_launcher.launchUrl(uri);
+}
```

Comment:
I'm not a big fan of this.
Passing around the BuildContext into utility functions.

I'd rather have this as an extension to the account itself.

Reply:
`Account.launchUrl(...)` doesn't make sense to me. Why is the Account launching the URL? We only need the Account to ensure the URL is valid which is a different thing.

Reply:
This is a bad timing for this kind of discussion.
In general, I'd say this should be handled by a bloc or for now in the widget itself.

It isn't obvious to the caller that this method needs access to the `Account` and therefore requires it to be in injected into the tree.
Maybe just make the account a parameter instead of the context and let the resolving happen outside this method.

We can discuss letting this is for now knowing that blocs will come to properly fix this.

Reply:
Ok, I'll go with the account as a parameter.